app/pages/home-for-user/index.jsx

```diff
@@ -70,7 +70,7 @@ export default class HomePageForUser extends React.Component {
   }
 
   getFeaturedProject() {
-    const query = { featured: true, launch_approved: true, cards: true };
+    const query = { featured: true, launch_approved: true };
```

Comment:
Won’t this remove the project avatar from the response, re-introducing the problem we had on the original PR, where the featured project image doesn’t load?

Reply:
Sorry, I'll add it back in there.